assets/differentialabundance_report.Rmd

```diff
@@ -139,59 +144,6 @@ library(DT)
 datatable(NULL)
 ```
 
-```{r, echo=FALSE}
```

Comment:
Reviewers: this is just because the function was moved up into shinyngs

Reply:
I appreciate such comments